packages/myst-directives/src/mermaid.ts

```diff
@@ -1,17 +1,21 @@
 import type { DirectiveSpec, DirectiveData, GenericNode } from 'myst-common';
+import { addCommonDirectiveOptions, commonDirectiveOptions } from './utils.js';
 
 export const mermaidDirective: DirectiveSpec = {
   name: 'mermaid',
+  options: {
+    ...commonDirectiveOptions('mermaid'),
```

Comment:
Should `mermaid` have enumerator? Are these currently treated as figures...?

Reply:
Changed to just label on mermaids!